Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
This PR is looking very complete. I have one question about the encoder direction. Does this make the contrast increase when the knob it turned clockwise? And, do the menus go in the __down__ direction when you turn the knob clockwise? The reason I ask is that we do want to add an option to reverse the direction that the knob moves through menus, for a knob to the right of the display. That change will only reverse menu movement. So I want to make sure that when we do that, reversing the knob direction, it works the same way, with clockwise moving in the __up__ direction in the menus.